target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -298,6 +298,12 @@ mzk-ex750np)
 na930)
 	set_usb_led "$boardname:blue:status"
 	;;
+netgear,r6120)
+	ucidef_set_led_default "power" "power" "$boardname:green:power" "1"
+	ucidef_set_led_netdev "lan" "lan" "$boardname:green:lan" "eth0.2"
```

Comment:
depending on the configuration, the lan ports might not be eth0.2. Please use `ucidef_set_led_switch` instead. the last argument is a bitmask of ports to trigger.

But if eth0.2 is your lan port something is wrong. According to your configuration, eth0.1 should be lan.

Reply:
@Brother-Lal this needs to be fixed before we can merge it.